lib/python/rosie/ws_client.py

```diff
@@ -170,6 +171,27 @@ def _get(self, method, return_ok_prefixes=False, **kwargs):
         if not request_details:
             raise RosieWSClientError(method, kwargs)
 
+        # Filter security warnings from urllib3 on python <2.7.9. Obviously, we
+        # want to upgrade, but some sites have to run cylc on platforms with
+        # python <2.7.9. On those platforms, these warnings serve no purpose
+        # except to annoy or confuse users.
+        if sys.version_info < (2, 7, 9):
+            import warnings
+            try:
+                from requests.packages.urllib3.exceptions import (
+                    InsecurePlatformWarning)
+            except ImportError:
+                pass
+            else:
+                warnings.simplefilter("ignore", InsecurePlatformWarning)
```

Comment:
In cylc/cylc#2688 the more primative `SecurityWarning` was used (which also covers `InsecureRequestWarning`), is this not needed here?

Reply:
I copied the latest code in Cylc. We don't need the `InsecureRequestWarning` here because we don't have the issue where we have to call the server without verifying its SSL certificate.